src/style.css

```diff
@@ -0,0 +1,89 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@600&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background-repeat: no-repeat;
+  text-decoration: none;
+  list-style: none;
+  font-family: 'inter', sans-serif;
+}
+
+.list {
+  display: flex;
+  justify-content: space-between;
+  padding: 2px 20px;
+  gap: 20px;
+  align-items: center;
+}
+
+.list-check {
+  display: flex;
+  justify-content: space-between;
+  gap: 20px;
+  align-items: center;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: rgb(7, 49, 49);
+}
+
+.main-Page {
+  background-color: #8f70d8;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  max-width: 100vw;
+  border-radius: 20%;
+  box-shadow: 2px 2px;
+}
+
+.tittle {
+  font-weight: 300;
+  font-size: 38px;
+  color: #fff;
+}
+
+.toDoList {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  gap: 1em;
+}
+
+.task {
+  width: 65%;
+  height: 80%;
+  border-radius: 5px;
+  font-size: 20px;
+}
+
+.add {
+  display: block;
+  border: none;
+  background-color: blue;
+  color: whitesmoke;
+  padding: 8px 15px;
+  font-size: 20px;
+  border-radius: 50%;
+}
+
+.listItem {
+  flex: 1;
+  gap: 10em;
+}
+
+.Completedbtn {
+  background-color: rgb(33, 19, 116);
+  color: #fff;
+  width: 40%;
+  margin-left: 30%;
+  margin-bottom: 50%;
+  padding: 8px 15px;
+  border-radius: 50%;
+  border-color: rgb(240, 52, 177);
```

Comment:
- [ ] Kindly make sure your todo matches the style in the screencast [video](https://www.youtube.com/watch?v=uUvp0XgE--s) in the requirements. Kindly check the difference

The template | Your design 
---|---
![image](https://user-images.githubusercontent.com/65068771/194517113-23044f28-27ff-4ac4-9c58-f603aad5413b.png) | ![image](https://user-images.githubusercontent.com/65068771/194516705-215f681d-06d2-4b7e-bb9d-9d582e40fcfc.png)